axon-cosmosdb-spring-boot-autoconfigure/src/test/resources/log4j2.properties

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (c) 2010-2022. Axon Framework
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+name=AxonTestConfiguration
+appenders=console
+appender.console.type=Console
+appender.console.name=STDOUT
+appender.console.layout.type=PatternLayout
+appender.console.layout.pattern=%d [%t] %-5p %-30.30c{1} %x - %m%n
+rootLogger.level=info
+rootLogger.appenderRefs=stdout
+rootLogger.appenderRef.stdout.ref=STDOUT
+logger.axon.name=org.axonframework
```

Comment:
Is this logger tuned towards _this_ module or is it a copy from another one? Changing log levels for the `AxonServerConnectionManager` and `TrackingEventProcessor` feel slightly off for the Cosmos extension.

Reply:
True, removed some others as well.